app/assets/javascripts/discourse/app/controllers/review-settings.js

```diff
@@ -25,4 +27,16 @@ export default Controller.extend({
         .finally(() => this.set("saving", false));
     },
   },
+
+  @discourseComputed("settings.reviewable_score_types")
+  scoreTypes() {
+    const username = I18n.t("review.example_username");
+    const types = this.get("settings.reviewable_score_types");
+
+    types.forEach((type) => {
+      type.title = type.title.replace("%{username}", username);
+    });
+
+    return types;
```

Comment:
```suggestion
    return types.map((type) => Object.assign({}, type, {
      title: type.title.replace("%{username}", username),
    }));
```

(you don't want to modify `settings.reviewable_score_types`)